capsules/src/led.rs

```diff
@@ -75,6 +75,9 @@ impl<'a, G: hil::gpio::Pin + hil::gpio::PinCtl> Driver for LED<'a, G> {
                 }
             }
 
+            // get number of LEDs
+            3 => pins.len() as isize,
+
```

Comment:
Might be better for this to be a `0` rather than a `3`. This pattern makes sense _at least_ for the buttons driver as well (probably also the GPIO driver), and `0` lets all of them use the same command number regardless of how many other commands they support.

I know it's a larger change because it's not backwards compatible, but are you willing to make it?